eth-contracts/contracts/GovernanceV2.sol

```diff
@@ -121,7 +122,7 @@ contract GovernanceV2 is InitializableV2 {
     /***** Proposal storage *****/
 
     /// @notice ID of most recently created proposal. Ids are monotonically increasing and 1-indexed.
-    uint256 lastProposalId = 0;
+    uint256 lastProposalId;
```

Comment:
yep exactly, this change is not *required*, but i figured it would be good to have. also this is how i tested if the contract was otherwise upgrade safe

Reply:
would like @hareeshnagaraj 's eyes on this one as well

Reply:
@dmanjunath - I don't think the value being set would result in it being overridden or not as it would be set in the new logic contract, not in storage. The only way it would get corrupted is if this value was removed or reordered. Does that sound right @SidSethi ?

Reply:
yeah @hareeshnagaraj thats my understanding as well